lua/wikis/commons/Infobox/Map.lua

```diff
@@ -105,7 +106,13 @@ end
 ---@param args table
 ---@return string?
 function Map:getGame(args)
-	return Game.name{game = args.game}
+	local game = Game.name{
+		game = args.game,
+		useDefault = Logic.nilOr(Logic.readBoolOrNil(args.useDefaultGame), true)
+	}
+	assert(game, 'Missing or invalid game specified')
```

Comment:
Do all wikis have a defaultGame so this won't error on wikis not setting specific games?

Reply:
yes, but some might not want to show/store game at all
(e.g. if the map is applicable for several)